src/recap.8

```diff
@@ -17,37 +17,37 @@
 .\" You should have received a copy of the GNU General Public
 .\" License along with this manual; if not, write to the Free
 .\" Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
-.\" Boston, MA 02110-1301 USA.
+.\" Boston, MA 02110\-1301 USA.
```

Comment:
I don't think this is needed, the license is never shown in the man page.

Reply:
Good catch. You're right, it's not needed. This got updated (in a batch update) because I did a search for all non-escaped dashes and escaped them.

I fixed them all, thanks!